src/config/default.properties

```diff
@@ -5,7 +5,7 @@ db_username=vimc
 db_password=changeme
 db_driver=org.postgresql.Driver
 
-testdb_image=docker.montagu.dide.ic.ac.uk:5000/montagu-db:master
+testdb_image=docker.montagu.dide.ic.ac.uk:5000/montagu-db
```

Comment:
This way the testdb_image is a valid identifier in its own right. I can see the advantages of both approaches; I think I'll leave it as is.